server/routes/victorops/tasks.py

```
@@ -0,0 +1,427 @@
+                source_alert_id=event_db_id,
+                alert_title=incident_title,
+                alert_service=service_name,
+                alert_severity=severity,
+                correlation_result=correlation_result,
+                alert_metadata=alert_metadata,
+                raw_payload=payload,
+                org_id=org_id,
+            )
+            conn.commit()
+            return True
+    except Exception as corr_exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)